src/android.ts

```diff
@@ -528,10 +532,10 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
                 throw new Error(`Invalid battery optimization state: ${state}`);
         }
     },
-    startApp(appId) {
+    async startApp(appId) {
         // We deliberately don't await these since objection doesn't exit after the app is started.
         if (options.capabilities.includes('certificate-pinning-bypass')) {
-            const process = execa('objection', [
+            const process = (await venv)('objection', [
```

Comment:
Why is it `venv` and not `python`?

Reply:
We don't want to await objection, but `python`, we can only await the whole thing.

Reply:
I see. A comment to clarify that would be nice.